src/test/Makefile-server.am

```diff
@@ -30,6 +30,10 @@ ceph_perf_objectstore_LDADD = $(LIBOS) $(UNITTEST_LDADD) $(CEPH_GLOBAL)
 ceph_perf_objectstore_CXXFLAGS = $(UNITTEST_CXXFLAGS)
 bin_DEBUGPROGRAMS += ceph_perf_objectstore
 
+ceph_perf_local_SOURCES = test/perf_local.cc test/perf_helper.cc
```

Comment:
@yuyuyu101 you might want to include perf_helper.h in `noinst_HEADERS` or `ceph_perf_local_SOURCES`.